Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierWebsiteQuestion.cshtml

```diff
@@ -1,60 +1,61 @@
+                            @ViewData["Title"]
+                        </h1>
+                    </legend>
+                    <div class="govuk-radios govuk-!-margin-top-6" data-module="govuk-radios">
+                        <div class="govuk-radios__item">
+                            <input class="govuk-radios__input" id="websiteReg" name="HasWebsiteAddress" type="radio"
+                                   value="true" aria-controls="conditional-website-yes" aria-expanded="false" @(Model.HasWebsiteAddress == true ? "checked" : "")>
+                            <label class="govuk-label govuk-radios__label" for="websiteReg">
+                                @StaticTextResource.Global_Yes
+                            </label>
+                        </div>
 
-                            <div class="govuk-radios__item">
-                                <input class="govuk-radios__input" id="websiteReg-2" name="HasWebsiteAddress" type="radio" value="false"
-                                @(Model.HasWebsiteAddress == false ? "checked" : "")>
-                                <label class="govuk-label govuk-radios__label" for="websiteReg-2">No</label>
+                        <div class="govuk-radios__conditional govuk-radios__conditional--hidden" id="conditional-website-yes">
+                            <div class="govuk-form-group @(websiteRegHasError ? "govuk-form-group--error" : "")">
+                                <label class="govuk-label" for="websiteAddress">
+                                    @StaticTextResource.Supplier_WebsiteQuestion_WebsiteAddress
+                                </label>
+                                <input class="govuk-input govuk-!-width-one-third @(websiteRegHasError ? "govuk-form-group--error" : "")"
+                                       id="websiteAddress" name="WebsiteAddress" spellcheck="false" autocomplete="false" aria-describedby="website-address-hint"
+                                       value="@(Model.HasWebsiteAddress == true ? Model.WebsiteAddress : "")">
                             </div>
+                        </div>
 
+                        <div class="govuk-radios__item">
+                            <input class="govuk-radios__input" id="websiteReg-2" name="HasWebsiteAddress" type="radio" value="false"
+                            @(Model.HasWebsiteAddress == false ? "checked" : "")>
+                            <label class="govuk-label govuk-radios__label" for="websiteReg-2">@StaticTextResource.Global_No</label>
                         </div>
 
-                    </fieldset>
-                </div>
+                    </div>
+
+                </fieldset>
+            </div>
 
-                <govuk-button class="govuk-!-margin-top-4">Save and continue</govuk-button>
-            </form>
-        </div>
+            <govuk-button class="govuk-!-margin-top-4">@StaticTextResource.Supplier_WebsiteQuestion_SaveAndContinue</govuk-button>
```

Comment:
There was another Save and continue somewhere else wasn't there. I think we should add a `Global_SaveAndContinue`

Reply:
You're right, switched to Global_SaveAndContinue